src/osd/PG.cc

```diff
@@ -3161,7 +3160,7 @@ bool PG::sched_scrub()
     return false;
   }
 
-  bool time_for_deep = (ceph_clock_now(cct) >
+  bool time_for_deep = (ceph_clock_now(cct) >=
```

Comment:
It is kinda confusing if *osd_deep_scrub_interval* and *osd_scrub_min_interval* have the same value, but not each time the scrub is a deep scrub..